source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -47,6 +50,34 @@ class EvaluationContext {
     set $data(value: any) {
         this._$data = value;
     }
+
+    generateVersionJson() {
```

Comment:
This may not work. Customer may use templating like this:
```
<script src="https://unpkg.com/adaptivecards-templating/dist/adaptivecards-templating.min.js"></script>
```

You may have to hard coded it. also verify the version is matching with what in package.json in CI pipeline.

Reply:
`$_acTemplateVersion` is working with scripts. I tested with the following files included in an html file:

```
<script src="https://unpkg.com/markdown-it@8.4.0/dist/markdown-it.js"></script>
<script src="https://unpkg.com/adaptivecards/dist/adaptivecards.min.js"></script>
<script type="text/javascript" src="browser.js"></script>
<script type="text/javascript" src="adaptivecards-templating.min.js"></script>
```